frontend/src/CommentView.js

```diff
@@ -4,85 +4,121 @@ import Home from "./Home";
 import ReactDOM from "react-dom";
 
 import React, { useState } from "react";
-import Collapsible from 'react-collapsible';
+import Collapsible from "react-collapsible";
 
-import { ReactComponent as ThumbsUp } from "./thumbs-up.svg";
-import { ReactComponent as ThumbsDown } from "./thumbs-down.svg";
+import { ReactComponent as ThumbsUp } from "./svg/thumbs-up.svg";
+import { ReactComponent as ThumbsDown } from "./svg/thumbs-down.svg";
 
-function CommentViewHeader(props){
-    return (
-        <div>
-            <div style={{fontSize: 20}}>{props.pin.title}</div>
-            <div>{props.pin.description}</div>
-            <br></br>
-            
-            <button onClick={goHome} style={{
-                            position: 'absolute',
-                            right: 0,
-                            top: 0,
-                        }}> X  </button>
-            <div>Comments:</div>
-        </div>
-    );
+function CommentViewHeader(props) {
+  return (
+    <div>
+      <div style={{ fontSize: 20 }}>{props.pin.title}</div>
+      <div>{props.pin.description}</div>
+      <br></br>
+
+      <button
+        onClick={goHome}
+        style={{
+          position: "absolute",
+          right: 0,
+          top: 0,
+        }}
+      >
+        {" "}
+        X{" "}
+      </button>
+      <div>Comments:</div>
+    </div>
+  );
 }
 
-function goHome(){//add browser router??
-    ReactDOM.render(<Home />, document.getElementById("root"));
+function goHome() {
+  //add browser router??
+  ReactDOM.render(<Home />, document.getElementById("root"));
 }
 
 function CommentViewBody(props) {
-    const _comments = props.commentsData.map( (text, index) => {
-        return(
-            <Collapsible key={index} trigger = "▼/▲" open = 'true'>
-                {text.description} <br></br>
-                <ThumbsUp title="Upvotes" onClick={() => props.upVote(index)}/> {text.upvotes} 
-                <ThumbsDown title="Downvotes" onClick={() => props.downVote(index)}/> {text.downvotes} 
-            </Collapsible>
-        );
-    });
-    return <div>{_comments}</div>;
+  const _comments = props.commentsData.map((text, index) => {
+    return (
+      <Collapsible key={index} trigger="▼/▲" open="true">
+        {text.description} <br></br>
+        <ThumbsUp title="Upvotes" onClick={() => props.upVote(index)} />{" "}
+        {text.upvotes}
+        <ThumbsDown
+          title="Downvotes"
+          onClick={() => props.downVote(index)}
+        />{" "}
+        {text.downvotes}
+      </Collapsible>
+    );
+  });
+  return <div>{_comments}</div>;
 }
 
-function CommentView(){
-    let pin = {title:'Study Bench', description:'Nice stone bench with shade from trees', upVotes:3, downVotes:4,pinType:'Study Location', color:'green'}
-    let commentz = [{description:"I go here all the time", downvotes: 10, upvotes: 50},{description:"I got my shoes dirty here, I don't like it", downvotes: 6, upvotes: 32}]
+function CommentView() {
+  let pin = {
```

Comment:
this should be replaced with a call to database to connect frontend and backend

Reply:
Agreed. We're going to be doing that in a future pull request on a different branch. We're just pushing this to main so we can mark our progress as this is working on its own.